pkg/devcontainer/run.go

```diff
@@ -106,6 +119,11 @@ func (r *Runner) prepare() (*config.SubstitutedConfig, *WorkspaceConfig, error)
 	if parsedConfig.WorkspaceMount != "" {
 		workspace.WorkspaceMount = parsedConfig.WorkspaceMount
 	}
+
+	if options.DevContainerImage != "" {
+		parsedConfig.ImageContainer = config.ImageContainer{Image: options.DevContainerImage}
```

Comment:
What happens when in the devcontainer.json a Dockerfile build is specified? Does this still work then?

Reply:
It was ignored, I've fixed it now :+1: